go/tao/keys.go

```diff
@@ -39,8 +39,8 @@ import (
 	"github.com/jlmucb/cloudproxy/go/tao/auth"
 	"github.com/jlmucb/cloudproxy/go/util"
 
-	"golang.org/x/crypto/hkdf"
-	"golang.org/x/crypto/pbkdf2"
+	"github.com/golang/crypto/hkdf"
+	"github.com/golang/crypto/pbkdf2"
```

Comment:
When I try to "go get", I get errors like this:

grace3 ~/cloudproxy/go$ go get ./...
package github.com/golang/crypto/hkdf: code in directory /home/kwalsh/go/share/src/github.com/golang/crypto/hkdf expects import "golang.org/x/crypto/hkdf"
package github.com/golang/crypto/pbkdf2: code in directory /home/kwalsh/go/share/src/github.com/golang/crypto/pbkdf2 expects import "golang.org/x/crypto/pbkdf2"
package github.com/golang/crypto/ssh: code in directory /home/kwalsh/go/share/src/github.com/golang/crypto/ssh expects import "golang.org/x/crypto/ssh"

I think this means we are meant to keep using the old import path, even though the code has moved.